tests/sql/moz-fx-data-shared-prod/braze_external/changed_users_v1/test_braze_changed_users/moz-fx-data-shared-prod.braze_external.users_previous_day_snapshot_v1.yaml

```
@@ -0,0 +1,44 @@
+# braze users
+---
+# user 1
+- external_id: user_1
+  email: user_1@mail.com
+  mailing_country: country_1
+  email_subscribe: subscribed
+  basket_token: token_1
+  email_lang: EN
+  create_timestamp: 2020-01-01 10:00:00+00:00
+  update_timestamp: 2021-01-01 14:00:00+00:00
+  fxa_id_sha256: fxa_user_1
+  has_fxa: true
+  fxa_primary_email: user_1_primary@mail.com
+  fxa_lang: DE
+  first_service: service_1
+# user 2
+- external_id: user_2
+  email: user_2@mail.com
+  mailing_country: country_1
+  email_subscribe: subscribed
+  basket_token: token_1
+  email_lang: EN
+  create_timestamp: 2020-01-01 10:00:00+00:00
+  update_timestamp: 2021-01-01 14:00:00+00:00
+  fxa_id_sha256: fxa_user_2
+  has_fxa: false
+  fxa_primary_email: user_2_primary@mail.com
+  fxa_lang: DE
+  first_service: service_1
+# user 3
+- external_id: user_3
+  email: user_3@mail.com
+  mailing_country: country_1
+  email_subscribe: subscribed
+  basket_token: token_3
+  email_lang: DE
+  create_timestamp: 2020-01-01 10:00:00+00:00
+  update_timestamp: 2021-01-01 14:00:00+00:00
+  fxa_id_sha256: fxa_user_3
+  has_fxa: true
+  fxa_primary_email: user_3_primary@mail.com
+  fxa_lang: DE
+  first_service: service_1
```

No new line is failing the CI
```suggestion
  first_service: service_1

```